TextToSpeech/impl/TTSSpeaker.cpp

```diff
@@ -673,20 +767,30 @@ void TTSSpeaker::createPipeline() {
         tts_url.append("&text=init");
         std::string LoopbackEndPoint = LOOPBACK_ENDPOINT;
```

Comment:
These two variables can be removed if not used

Reply:
ok